source/_integrations/isy994.markdown

```diff
@@ -231,9 +234,11 @@ Send an ISY set_ramp_rate command to a `light` Node to set the devices' ramp rat
 | `entity_id`            |     no   | Name(s) of target entities for the command, e.g., `light.front_porch`. |
 | `value` | no | The integer index value to set the Ramp Rate to in a range of `0` (9.5 minutes) to `31` (0.1 Seconds), e.g., `28` |
 
-#### Service `isy994.system_query`
+#### Service `isy994.system_query` (Deprecated)
```

Comment:
Deprecated services should be removed from the documentation

Reply:
Updated.